docs/development/releasing.md

```diff
@@ -230,4 +230,34 @@ Then edit the release on GitGub:
 
 1. Remove `beta` from the release name: electron v1.7.5 ~~beta~~
 1. Uncheck the `prerelease` checkbox.
-1. Click "Update release" 
\ No newline at end of file
+1. Click "Update release"
+
+## Fix missing binaries of a release manually
+
+In the case of a corrupted release with broken CI machines, we might have to
+re-upload the binaries for a already published release.
+
+The first step is to go to the Releases page and delete the corrupted binaries
```

Comment:
`The first step is to go to the [Releases](https://github.com/electron/electron/releases) page`